sparta/sparta/ports/PortSet.hpp

```diff
@@ -47,7 +47,7 @@ namespace sparta
          * \return Pointer to the port or nullptr if not found
          */
         Port * getPort(const std::string & named_port) const {
-            for(uint32_t i = 0; i < Port::N_DIRECTIONS; ++i) {
+            for(uint32_t i = 0; i <Port::N_DIRECTIONS; ++i) {
```

Comment:
Magic space disappears and it could be `Port::Direction::N_DIRECTIONS`

Reply:
Fixed.